rust/basic_bitcoin/README.md

```diff
@@ -1,225 +1,177 @@
 # Basic Bitcoin
 
-This tutorial will walk you through how to deploy a sample [canister smart contract](https://internetcomputer.org/docs/current/developer-docs/multi-chain/bitcoin/overview) **that can send and receive Bitcoin** on the Internet Computer.
+This example demonstrates how to deploy a smart contract on the Internet Computer that can receive and send Bitcoin, including support for legacy (P2PKH), SegWit (P2WPKH), and Taproot (P2TR) address types.
 
 ## Architecture
 
-This example internally leverages the [ECDSA
-API](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-ecdsa_public_key),
-[Schnorr
-API](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-sign_with_schnorr),
-and [Bitcoin
-API](https://github.com/dfinity/bitcoin-canister/blob/master/INTERFACE_SPECIFICATION.md)
-of the Internet Computer.
+This example integrates with the Internet Computer's built-in:
 
-For a deeper understanding of the ICP < > BTC integration, see the [Bitcoin integration documentation](https://wiki.internetcomputer.org/wiki/Bitcoin_Integration).
+- [ECDSA API](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-ecdsa_public_key)
+- [Schnorr API](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-sign_with_schnorr)
+- [Bitcoin API](https://github.com/dfinity/bitcoin-canister/blob/master/INTERFACE_SPECIFICATION.md)
 
-## Prerequisites
+For background on the ICP <-> BTC integration, refer to the [Bitcoin integration documentation](https://wiki.internetcomputer.org/wiki/Bitcoin_Integration).
```

Comment:
did you align on @jessiemongeon1 with this? wondering if we'd want to prefer to refer to the learn hub 🤔 

```suggestion
For background on the ICP <-> BTC integration, refer to the [Learn Hub](https://learn.internetcomputer.org/hc/en-us/articles/34211154520084-Bitcoin-Integration).
```